doc/cephadm/adoption.rst

```diff
@@ -118,6 +118,12 @@ Adoption process
      document for instructions that describe how to import existing
      ssh keys.
 
+   .. note::
+     It is also possible to have cephadm use a non root user to ssh
```

Comment:
```suggestion
     It is also possible to have cephadm use a non-root user to ssh
```
Not a big deal but this is how we say "non root" in the section you link to. Only fix if you feel like it.